lib/events/api.go

```diff
@@ -456,6 +456,9 @@ const (
 	// WindowsDesktopSessionEndEvent is emitted when a user  disconnects
 	// from a desktop.
 	WindowsDesktopSessionEndEvent = "windows.desktop.session.end"
+
+	// UserCertsEmitEvent is emitted when a user certificate set is emitted.
+	UserCertsEmitEvent = "certs.user.emit"
```

Comment:
Maybe:

```suggestion
	UserCertsEmitEvent = "user.certs.emitted"
```

Reply:
I don't know about `.emit` vs. `.emitted`, there's plenty of `.create`/`.update`/`.delete` (but also plenty of (`.created`/`.deleted`).

I was thinking that we might want to also add an event for non-user certificate issuance in the future, but having it as part of the `user` event group would also make sense (with a `T10xxI` code to match).

Reply:
Fair enough about `.emit`. I'm happy with whatever you prefer.

Reply:
How about `user.cert.create` seems to line up with all the other verbs we use (create) and describes what is happening.